app/src/androidTest/java/com/rootstrap/android/tests/SignInActivityTest.kt

```diff
@@ -19,14 +19,16 @@ import org.junit.Test
 @HiltAndroidTest
 class SignInActivityTest : BaseTests() {
```

Comment:
Maybe we should change the name of this class to OnBoardingActivity as it now validates if the OnBoardingActivity is launched successfully